qiskit/test/decorators.py

```diff
@@ -55,14 +54,7 @@ def is_aer_provider_available():
     Returns:
         bool: True if simulator executable is available
     """
-    # TODO: HACK FROM THE DEPTHS OF DESPAIR AS AER DOES NOT WORK ON MAC
```

Comment:
How long as this been here? Aer has worked on mac for a really long time now...

Reply:
```
commit b2bbb3f05968f3cadc2993ed66123077d80026ef
Author: Jay M. Gambetta <jay.gambetta@us.ibm.com>
Date:   Fri Dec 28 17:32:27 2018 -0500

    Remove legacy simulators  (#1615)
```

so not too long, you know